scripts/packages/bazel.sh

```diff
@@ -203,11 +203,11 @@ if [[ ! -x $BAZEL_REAL ]]; then
     if [[ -x $(command -v curl) && -w $wrapper_dir ]]; then
       (echo ""
       echo "You can download the required version directly using this command:"
-      echo "  (cd \"${wrapper_dir}\" && curl -fLO https://releases.bazel.build/${bazel_version}/release/${long_binary_name} && chmod +x ${long_binary_name})") 2>&1
+      echo "  pushd `pwd` && (cd \"${wrapper_dir}\" && curl -fLO https://releases.bazel.build/${bazel_version}/release/${long_binary_name} && chmod +x ${long_binary_name}) && popd") 2>&1
```

Comment:
The surrounding `(` `)` should already make sure that this is executed in a subshell and you end up where you started.
Isn't that the case for you?

Reply:
Indeed this was the intent behind this. Does it not work?

Reply:
yeah, it does not work for me, I am using Mac and bash shell.

Reply:
I did manage cd before, surprising that no one had the similar issue.